tests/components/sfr_box/conftest.py

```diff
@@ -79,10 +79,17 @@ def ftth_get_info() -> Generator[FtthInfo, None, None]:
         yield info
 
 
+@pytest.fixture(params=["adsl"])
+def net_infra(request: pytest.FixtureRequest) -> str:
+    """Fixture to parametrize net_infra in SFRBox.system_get_info."""
+    return request.param
```

Comment:
```suggestion
@pytest.fixture
def net_infra() -> str:
    """Fixture to parametrize net_infra in SFRBox.system_get_info."""
    return "adsl"
```

There is no need to fiddle with the fixture requests for these things?